src/components/dialog-full-screen/dialog-full-screen.style.js

```diff
@@ -11,13 +11,15 @@ import {
 import { StyledForm } from "../form/form.style";
 
 const StyledDialogFullScreen = styled.div`
-  background-color: ${({ theme }) => theme.disabled.input};
+  background-color: var(--colorsUtilityMajor025);
   height: 100%;
   left: 0;
   position: fixed;
   top: 0;
   width: 100%;
-  z-index: ${({ theme }) => theme.zIndex.fullScreenModal};
+  z-index: ${({ theme }) =>
+    theme.zIndex
+      .fullScreenModal}; // TODO (tokens): implement elevation tokens - FE-4437
```

Comment:
Personally I think we should be adding these kind of reminders in the ticket description on our backlog, so they're all in one place and aren't public-facing

Reply:
I've added to the ticket description this file needs changing :)

Reply:
There is already few comments like that in code base. We can delete them all if needed. There's no need to add files to task - we can search through all code base

Reply:
This is only a minor thing, so if several comments exist currently then that should be fine. Personally though going forward, I think we should be placing any written reminders for future work along with the respective ticket.